packages/voorbeeld-design-tokens/documentation/color-sample/utrecht-color-sample.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ColorSample } from '@utrecht/component-library-react/dist/css-module';
+
+const meta = {
+  id: 'utecht-color-sample',
+  title: 'Components/ColorSample/Utrecht',
+  component: ColorSample,
```

Comment:
Ik zou nog even `{ args: { color: "#74295F" } },` ofzo toevoegen, zodat er een kleur zichtbaar is.